core/src/main/java/org/ehcache/statistics/StoreOperationOutcomes.java

```diff
@@ -22,38 +22,24 @@
  */
 public interface StoreOperationOutcomes {
   /**
-   * The Enum GetOutcome.
+   * The access outcomes
    */
-  enum GetOutcome {
-
-    /** hit. */
+  enum AccessOutcome implements StoreOperationOutcomes {
     HIT,
-    /** miss. */
-    MISS;
+    MISS,
+    WRITE,
+    REMOVE
   }
 
-  /**
-   * The Enum PutOutcome.
-   */
-  enum PutOutcome {
-    /** added. */
-    ADDED,
-    /** updated. */
-    UPDATED
-  };
-
-  /**
-   * The Enum RemoveOutcome.
-   */
-  enum RemoveOutcome {
-    /** success. */
-    SUCCESS
-  };
+  enum TierEfficiencyOutcome implements StoreOperationOutcomes {
```

Comment:
Efficiency seems like a weird name.  Efficiency would be something derived from this raw data right?  TierUtilityOutcome?  TierHelpfullnessOutcome?